src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -202,25 +202,34 @@ export const ConsultationDetails = (props: any) => {
                   </div>
                 )}
               </div>
-              <div className="flex flex-col mt-2">
-                <div className="text-sm text-gray-700">
-                  Created on{" "}
-                  {moment(consultationData.created_date).format("lll")}
+              <div className="flex flex-col mt-2 space-y-1">
+                <div className="text-sm text-gray-700 font-semibold leading-relaxed">
+                  <span className="text-gray-900">Created: </span>
+                  {moment(consultationData.created_date).format("lll")} |
                   {consultationData.created_by && (
-                    <span>
-                      {`by ${consultationData.created_by?.first_name} ${consultationData.created_by?.last_name} @${consultationData.created_by?.username} (${consultationData.created_by?.user_type})`}
-                    </span>
+                    <div className="block md:inline">
+                      <span className="font-bold">
+                        {` ${consultationData.created_by?.first_name} ${consultationData.created_by?.last_name}  `}
+                      </span>
+                      <span className="badge badge-pill badge-warning">
+                        {`@${consultationData.created_by?.username} (${consultationData.created_by?.user_type})`}
+                      </span>
+                    </div>
                   )}
                 </div>
 
-                <div className="text-sm text-gray-700">
-                  Last Modified on{" "}
-                  {moment(consultationData.modified_date).format("lll")}{" "}
+                <div className="text-sm text-gray-700 font-semibold leading-relaxed">
+                  <span className="text-gray-900">Last Modified: </span>
```

Comment:
One suggestion, no need to have text `Last modified on` in the component. Use [`FA's history icon`](https://fontawesome.com/v5.15/icons/history?style=solid). That will make much clearer and minimize text. 

This icon is already added in facility list page, you can add your own creativity. 
My view i think will be best if  `created` and `updated` dates to show in flex with left right respectively

Reply:
Here, won't it be easier to understand if with the text instead of the icon?

Reply:
![image](https://user-images.githubusercontent.com/52771399/128540382-2ab322fa-17c9-43c8-84a4-93810f58546b.png)

i was thinking like that but in right side. 
Actually it would be good to use icon i think to optimize of mobile view. because text `Created` and `Updated` takes enough space than actual information like date and by user. Also i think these dates should be shown in littlebit more small text(`text-xs` may be)

Reply:
btw not much need of it. UI already optimized for mobile view